pkgs/development/python-modules/digitalocean/default.nix

```diff
@@ -1,23 +1,54 @@
-{ stdenv, buildPythonPackage, fetchPypi, requests, jsonpickle }:
+{ buildPythonPackage
+, fetchFromGitHub
+, fetchPypi
+, isPy3k
+, jsonpickle
+, mock
+, pytest
+, pytestCheckHook
+, requests
+, responses
+, stdenv
+}:
 
 buildPythonPackage rec {
   pname = "python-digitalocean";
-  version = "1.13.2";
+  version = "1.15.0";
 
-  src = fetchPypi {
-    inherit pname version;
-    sha256 = "0h4drpdsmk0b3rlvg6q6cz11k23w0swj1iddk7xdcw4m7r7c52kw";
+  src = fetchFromGitHub {
+    name = "python-digitalocean-${version}";
```

Comment:
you shouldn't set the derivation here?
```suggestion
```

Reply:
@cleverca22 said it makes debugging easier

Reply:
I believe it's very intentional that the default name is `source` in the fetchers.
See https://github.com/NixOS/nixpkgs/commit/c3255fe8ec326d2c8fe9462d49ed83aa64d3e68f. If it makes debugging easier, you can just change it when debugging.

Reply:
I think what they meant was, it's impossible to have a bad fixed-point derivation. Having the name as "source" can create an instance where you change the version of the out package, but the fetchurl is still retrieving the old outputs because there's a cached instance. Passing an explicit name will create a new .drv and there will be a hash-mismatch if you forget to then update the sha256

Reply:
I believe https://github.com/NixOS/nixpkgs/commit/c3255fe8ec326d2c8fe9462d49ed83aa64d3e68f was an attempt to standardize the fetcher derivation names, not make it more ergonomic

Reply:
yeah what jon said :+1:

Reply:
If we feel so strongly about changing this for fetchers derivations, this PR isn't the place to spread this. All it does is confuse contributors and divide our codebase. If we think https://github.com/NixOS/nixpkgs/commit/c3255fe8ec326d2c8fe9462d49ed83aa64d3e68f wasn't a good choice, let's change that everywhere, not just here.

Reply:
@jonringer Would you care to open an issue referencing our discussion?